alembic/versions/9e208dfd0778_add_correlator_config_table.py

```diff
@@ -0,0 +1,33 @@
+"""Add correlator config table
+
+Revision ID: 9e208dfd0778
+Revises: 7463268309ab
+Create Date: 2020-10-07 05:01:49.357142+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '9e208dfd0778'
+down_revision = '7463268309ab'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
This and similar lines should be removed -- I'm sure that you made changes beyond the auto generated commands.